.github/workflows/vision.yaml

```diff
@@ -0,0 +1,68 @@
+name: vision
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'vision/**'
+  pull_request:
+    paths:
```

Comment:
Please add `vision/productSearch/**` to `paths-ignore` so changes to vision-productSearch don't trigger tests of vision.

Reply:
Where would I find the definition for this YAML schema? I've made the change to the YAML file but I don't think it's correct ....

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore